apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsTransactionHelper.java

```diff
@@ -47,15 +47,16 @@ public JaxRsTransactionHelper(ElasticApmTracer tracer) {
     public void setTransactionName(@Nonnull Transaction currentTransaction,
                                    @Nonnull String signature,
                                    @Nullable String pathAnnotationValue)  {
-        System.out.println("Signature = " + signature);
+        String transactionName = "";
         if (signature != null) {
```

Comment:
`signature` is annotated with `@Nonnull` so a null check seems unnecessary.

Reply:
fixed